src/main/java/com/dev/cinema/dto/UserDto.java

```diff
@@ -0,0 +1,10 @@
+package com.dev.cinema.dto;
+
+import lombok.Data;
+
+@Data
+public class UserDto {
+    private Long id;
```

Comment:
do we need this field?

Reply:
We have the method get by id so we could use this field for that case. We could receive requests get by id for example.

Reply:
let's remove this field

if we would like to find user by id, we would not use DTO
we would rather pass the id as @PathVariable or @RequestParam

Reply:
Removed. Could I use this UserDto as RequestUserDto and add one more ResponceUserDto with id as well and send this response with id in the future hw?